index.js

```diff
@@ -126,7 +126,7 @@ module.exports.errorLogger = function (opts) {
                 'response-time': responseTime,
                 "response-hrtime": hrtime,
                 "status-code": status,
-                'req-headers': req.headers,
+                'req-headers': JSON.parse(JSON.stringify(req.headers)),
```

Comment:
Ugh. I don't like to merge that. This has a huge performance impact.
Can you try to use `Object.create(req.headers)` or Object.assign({}, req.headers)` instead?